### PR TITLE
Add Offline QR Code Generator

### PR DESCRIPTION
**Pull request by rugk**

### CODE REVIEW

File: _data/projects/offlineqr.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Offline QR Code Generator
+desc: Browser add-on allowing you to quickly generate a QR code offline with the URL of the open tab or other text!
+site: https://github.com/rugk/offline-qr-code
+tags:
+- oss
+- web
+- javascript
+upforgrabs:
+  name: good first issue
+link: https://github.com/rugk/offline-qr-code/labels/good%20first%20issue
```
Comment (ritwik12) [inline _data/projects/offlineqr.yml]: @rugk Please give space before "link:", build is failing due to that.

**Comment by ritwik12** [comment]
@rugk Thanks!!

**Comment by rugk** [comment]
BTW I did not get a notification that you added this label, so if I had not checked this PR again manually I would have never been notified/made aware by GitHub that something is to do here and would likely forget this PR.

So maybe consider at least commenting (or using the review feature of GitHub…? I think that triggers a notification.), so contributors get a notification.

**Comment by ritwik12** [comment]
@rugk I have already created a review and also pointed out what to do. I guess you missed the notification,

File: _data/projects/offlineqr.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/offlineqr.yml]: @rugk I already mentioned you here.

**Comment by ritwik12** [comment]
capture

@rugk 
This was the review. Please let me know if you missed the notification or haven't got it so that I can take care of it in future.

**Comment by rugk** [comment]
Your review was still "pending". See all the yellow messages in your screenshot!

It has not been submitted yet, so I could not receive the messages. You need to submit your review first (in the code view, top right button), before it is shown to other users.

**Comment by ritwik12** [comment]
@rugk Apologies, I think I forgot to press submit button. Thanks for coming back and checking manually. I will take care of this in future.

File: _data/projects/offlineqr.yml (hunk 1 above)
Comment (rugk) [reply _data/projects/offlineqr.yml]: Now I can see that. :smile:

**Pull request merged**